dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HadoopUtils.java

```diff
@@ -129,6 +129,11 @@ private void init() throws NullPointerException {
             }
 
             String defaultFS = configuration.get(Constants.FS_DEFAULT_FS);
+
+            if (defaultFS == null) {
```

Comment:
Can we optimize line 139 - 153? Some code is duplicate.

Reply:
> Can we optimize line 139 - 153? Some code is duplicate.

👍 WIP.

Reply:
Hi @ruanwenjun PTAL.